wire-ios-request-strategy/WireRequestStrategy.xcodeproj/project.pbxproj

```diff
@@ -3170,6 +3171,10 @@
 /* End XCConfigurationList section */
 
 /* Begin XCLocalSwiftPackageReference section */
+		E67F36282BFDE35000B83E0F /* XCLocalSwiftPackageReference "../Packages/SourceryPlugin" */ = {
```

Comment:
Aren't these changes which you just made for demo purposes in our call?

Reply:
Right, thanks! removed it with 1686dc59efa741ea1aac79cbd63012a4bf7e3da4